projects/UCI_Sheng.yaml

```
@@ -0,0 +1,6 @@
+Department: Paul Merage School of Business
+Description: 'Research on the US housing market and transactions with the aim of comprehending
+  how housing prices move and are correlated with each other. '
+FieldOfScience: Economics
+Organization: University of California Irvine
```

```suggestion
Organization: University of California, Irvine
```

This org already exists, just missing a comma